src/main/java/Parser.java

```diff
@@ -0,0 +1,120 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * The Parser class deals with making sense of the user's command.
+ */
+public class Parser {
+    /**
+     * Gets a Command object
+     *
+     * @param command A string representing the user's command.
+     * @return A Command object.
+     */
```

Comment:
Is there a need to create a Parser in this case? Since it directly returns a Command object without much modification